File: InheritanceWith_base/InheritanceWith_base/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InheritanceWith_base
{
    class Person
    {
        public Person(string name)
        {
            m_strName = name;
        }

        public string GetName()
        {
            return m_strName;
        }

        private string m_strName;
    }
}

File: JohnBryce/DesignPattern/Iterator/Project1/Project1/Person.cpp
#include "Person.h"



Person::Person(string strName, int nAge) : m_strName(strName), m_nAge(nAge)
{
	
}


Person::~Person()
{
}

const string & Person::GetName() const
{
	return m_strName;
}

int Person::GetAge() const
{
	return m_nAge;
}
